lib/ansible/modules/crypto/openssh_keypair.py

```diff
@@ -134,6 +139,7 @@ def __init__(self, module):
         self.check_mode = module.check_mode
         self.privatekey = None
         self.fingerprint = {}
+        self.public_key = {}
```

Comment:
Not sure why this and the one above it start out as an empty dict, but you are matching the surrounding code...

Reply:
Yes, that's the idea.
Thanks.